scripts/aws/monitor.py

```diff
@@ -201,6 +223,7 @@ def add_cloudwatchlog_role_to_apigateway(client, role_arn):
     parser.add_argument("--lambda-timeout", type=int, required=True)
     parser.add_argument("--lambda-memory", type=int, required=True)
     parser.add_argument("--kms-key", required=True)
+    parser.add_argument("--logging-level", required=True)
```

Comment:
Does it make sense to have this and timeout and memory have defaults to simplify calling the script?   I think we'd like the flexibility to change them, but would expect that we mostly would not need to.